scripts/run-tests.sh

```diff
@@ -12,8 +13,15 @@ GO_LDFLAGS="-X github.com/infinispan/infinispan-operator/launcher.Version=${VERS
 
 go clean -testcache ./test/e2e
 if [ -z "${TEST_NAME}" ]; then
-  go test -v ./test/e2e/"${TEST_BUNDLE}" -timeout 90m -ldflags "${GO_LDFLAGS}" -parallel "${PARALLEL_COUNT}"
+  go test -v ./test/e2e/"${TEST_BUNDLE}" -timeout ${TIMEOUT} -ldflags "${GO_LDFLAGS}" -parallel "${PARALLEL_COUNT}" 2>&1 | tee /tmp/testOutput
+  testExitCode=${PIPESTATUS[0]}
+  if [ ! -z ${TEST_REPORT_DIR} ]; then
+    mkdir -p ${TEST_REPORT_DIR}
+    cat /tmp/testOutput | ${GO_JUNIT_REPORT} > ${TEST_REPORT_DIR}/${TEST_BUNDLE}.xml
+  fi
+  rm /tmp/testOutput
+  exit $testExitCode
```

Comment:
I'm not sure if this is a good idea. if a test fails in `main`, for example, no other stages are run.

Reply:
That's already the case. I was maintaining this behaviour as it allows the CI to fail-fast and free up one of the two agents for other operator builds.